pkg/minikube/cluster/cluster.go

```diff
@@ -371,16 +373,32 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 }
 
 // GetHostLogs gets the openshift logs of the host VM.
-func GetHostLogs(api libmachine.API) (string, error) {
+// If follow is specified, it will tail the logs
+func GetHostLogs(api libmachine.API, follow bool) (string, error) {
```

Comment:
I know you have not added this method, but what is it doing here in _cluster.go_? And why is it called _GetHostLogs_? Shouldn't this really be in _pkg/minishift/openshift_?